libs/domains/cloud-providers/feature/src/lib/hooks/use-cloud-provider-database-instance-types/use-cloud-provider-database-instance-types.tsx

```diff
@@ -0,0 +1,21 @@
+import { useQuery } from '@tanstack/react-query'
+import { type CloudProviderEnum } from 'qovery-typescript-axios'
+import { queries } from '@qovery/state/util-queries'
+
+export interface UseCloudProviderDatabaseInstanceTypesProps {
+  cloudProvider: Extract<CloudProviderEnum, 'AWS' | 'SCW'>
+  databaseType: string
+  region?: string
+}
+
+export function useCloudProviderDatabaseInstanceTypes({
```

Comment:
Once region is enforced by typing you can reuse the data-access param (cf. `useCloudProviderInstanceTypes`)